meerk40t/core/elements.py

```diff
@@ -4520,18 +4668,18 @@ def reset_user_changes(node, copies=1, **kwgs):
             lambda node: isinstance(node.object, Shape)
             and not isinstance(node.object, Path)
         )
-        @self.tree_operation(_("Convert To Path"), node_type=("elem",), help="")
+        @self.tree_operation(_("Convert to path"), node_type=("elem",), help="")
         def convert_to_path(node, copies=1, **kwgs):
             node.replace_object(abs(Path(node.object)))
             node.altered()
 
         @self.tree_conditional(lambda node: isinstance(node.object, SVGElement))
         @self.tree_conditional_try(lambda node: not node.object.lock)
         @self.tree_submenu(_("Scale"))
-        @self.tree_iterate("scale", 1, 25)
+        @self.tree_iterate("scale", 25, 1, -1)
         @self.tree_calc("scale_percent", lambda i: "%0.f" % (600.0 / float(i)))
         @self.tree_operation(
-            _("Scale {scale_percent}%"), node_type="elem", help="Scale Element"
+            _("Scale %s%%") % "{scale_percent}", node_type="elem", help="Scale Element"
         )
```

Comment:
Hm. I thought this was silly, but I see the need to keep that bit of code away from the translators since it's hardcoded functional.

Reply:
Actually I think I was wrong in concept - I think the translator would translate "Scale {scale_percent}" to (say) "Size {scale_percent}" and the variable substitution would happen afterwards. (I was thinking of `_("Scale %s%" % scale_percent)` vs. `_("Scale %s%") % scale_percent` when I was thinking that this would be an issue.)